docs/installation/archlinux.md

```diff
@@ -66,6 +66,27 @@ If you need to add an HTTP Proxy, set a different directory or partition for the
 Docker runtime files, or make other customizations, read our systemd article to
 learn how to [customize your systemd Docker daemon options](/articles/systemd/).
 
+## Running docker with a manually defined network
```

Comment:
Awesome, that is just so much better.

Yeah, this doc thing, I don't do it too much and I'm afraid it shows :smile:

Changing it now.